src/caffeine/program_termination_s.f90

```diff
@@ -106,13 +106,13 @@ subroutine prif_error_stop_integer(quiet, stop_code)
 
     if (present(stop_code)) then
       if (.not.quiet) then
-        write(error_unit) "ERROR STOP ", stop_code
+        write(error_unit,'(a)') "ERROR STOP ", stop_code
```

Comment:
Shouldn't this format statement also include an entry for `stop_code`?

```suggestion
        write(error_unit,'(A, I5)') "ERROR STOP ", stop_code
```

I'm probably missing something here.

Reply:
@bonachea good catch.  I'm switching `I5` to `I0`, which lets the compiler select the `integer` width. 
```suggestion
        write(error_unit,'(A, I0)') "ERROR STOP ", stop_code
```